app/controllers/coordinator_controller.rb

```diff
@@ -2,9 +2,19 @@ class CoordinatorController < ApplicationController
   before_action :auth
 
   def deliver
+    current_deliveries = Transaction.includes({recipient: [:recipient_profile]}, :donation).where(delivered_at: nil)
```

Comment:
Line is too long. [116/80]<br>Space inside { missing.<br>Space inside } missing.